src/utilities/ticket-channel.go.tmpl

```diff
@@ -0,0 +1,13 @@
+{{/*
+	Takes all messages sent in a specified channel, creates a ticket owned by the user who sent it and witht the message as the reason, and deletes the original message.
```

Comment:
```suggestion
	Creates a ticket owned by the user who sent it and with the message as the reason, and deletes the original message.
```
Please adhere to the correct spelling. Moreover, this code does **not** take all messages in a channel, so please remove that.

Yes, the trigger causes this behaviour, but it's not the code. The leading comment shall describe the code in a neutral manner, with no regards to the trigger, or trigger type.

Moreover, this phrasing could cause confusion for end-users, as it might imply that it **moves** (as in, take elsewhere) all messages in that channel. Consider rephrasing it to be more precise.